docs/user-guide/how_to_use_svc_plugin.md

```diff
@@ -0,0 +1,332 @@
+# Volcano Job Plugin -- SVC User Guidance
+
+## Background
+**SVC Plugin** is designed for the communication for pods within a volcano job, which is essential for workloads such as
```

Comment:
what's this `SVC` short for? A full name added would be great for beginners

Reply:
It is short for `service`, a common short name for K8s users.

Reply:
I see, thanks for clarifying.